plugins/services/src/js/containers/service-connection/ServiceConnectionContainer.js

```diff
@@ -0,0 +1,137 @@
+import React from "react";
+
+import ConfigurationMap from "#SRC/js/components/ConfigurationMap";
+import ConfigurationMapHeading
+  from "#SRC/js/components/ConfigurationMapHeading";
+import ConfigurationMapLabel from "#SRC/js/components/ConfigurationMapLabel";
+import ConfigurationMapRow from "#SRC/js/components/ConfigurationMapRow";
+import ConfigurationMapSection
+  from "#SRC/js/components/ConfigurationMapSection";
+import ConfigurationMapValue from "#SRC/js/components/ConfigurationMapValue";
+import AlertPanel from "#SRC/js/components/AlertPanel";
+import AlertPanelHeader from "#SRC/js/components/AlertPanelHeader";
+import Service from "../../structs/Service";
+import ServiceConfigUtil from "../../utils/ServiceConfigUtil";
+import { getDisplayValue } from "../../utils/ServiceConfigDisplayUtil";
+
+class ServiceConnectionContainer extends React.Component {
+  getProtocolValue(portDefinition) {
+    let protocol = portDefinition.protocol || "";
+    protocol = protocol.replace(/,\s*/g, ", ");
+
+    return (
+      <ConfigurationMapValue>{getDisplayValue(protocol)}</ConfigurationMapValue>
+    );
+  }
+
+  getHostPortValue(portDefinition) {
```

Comment:
The service container is not related to the store, the store was created because the SDKEndpoints need another source for their values. This file is similar to "ServiceNetworkingConfigSection.js" (lines 90-142) that for each column has some presentation logic